ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -18,8 +20,22 @@ export default function AdvancedGasControls({
 
   return (
     <div className="advanced-gas-controls">
-      <FormField
-        titleText={t('gasLimit')}
+      <Box
+        display={Display.Flex}
+        alignItems={AlignItems.center}
+        marginBottom={1}
+      >
+        <Label htmlFor="gas-limit" marginRight={1}>
+          Gas limit
+        </Label>
+        <InfoTooltip contentText={t('editGasLimitTooltip')} position="bottom" />
```

Comment:
there have been visual changes to the title as we're using the `Label` component now, can you help fix this?

Reply:
Hey @dhruvv173, there may be some slight visual changes that align with the newer components which is correct. It should be mostly font-size the layout should remain the same. I haven't checkout out this branch yet but if you are noticing some font-size/font-weight changes that should be ok. Will try to review early this week

Reply:
Yes the concern was related to changes in font size/weight. Will wait for your review, thanks!